language-translator/src/main/java/com/ibm/watson/developer_cloud/language_translator/v2/model/CreateModelOptions.java

```diff
@@ -25,11 +28,11 @@
   private String baseModelId;
   private String name;
   private InputStream forcedGlossary;
+  private String forcedGlossaryFilename;
   private InputStream parallelCorpus;
+  private String parallelCorpusFilename;
   private InputStream monolingualCorpus;
-  private String forcedGlossaryContentType;
-  private String parallelCorpusContentType;
-  private String monolingualCorpusContentType;
```

Comment:
So is this something you think we need to add back? Otherwise I'll check to see if it's a breaking change or not.

Reply:
I think this is probably a breaking change, but I doubt it will affect anyone if we remove it.  My vote would be to leave it out, and only add them back if someone complains (and can't be convinced that they don't really need them).